ruby/vampires.rb

```diff
@@ -5,8 +5,14 @@ def interview_test
 puts "Welcome to the interview\nplease be seated!"
 puts "What is your name?"
 person_name = gets.chomp
-puts "How old are you?"
+age = 0 
+until (age > 15 && age <= 80)
+puts "How old are you? (between 15 to 80)"
 age = gets.chomp.to_i
+if age < 15 || age > 80
+  puts "Invalid age number."
+end
+end
 puts "What year were you born?"
 year = gets.chomp.to_i
 current_year = Time.new.year
```

Comment:
For vamp_test method

Consider the optimum condition when running your code.  What can we do to run only necessary processes?  We know that if the employee has Drac or Fang in their name then they are definitely a vampire.  If that is the case, do we need to run any of the other checks?

Consider:

```
if name_matches_vampire_params
  "Definitely a vampire"
else
  if #other checks
    "Something"
  elsif #more conditions...

  else
  end
end
```